File: 프로그래머스/0/120890. 가까운 수/가까운 수.py
def solution(array, n):
    diff = 101
    answer = 0
    for i in array:
        if abs(n-i) < diff:
            diff = abs(n-i)
            answer = i
        elif abs(n-i) == diff:
            answer = min(i, answer)
    
    return answer

File: 프로그래머스/0/120890. 가까운 수/가까운 수.js
function solution(array, n) {
    let minDiff = Infinity;
    let answer;
    array = array.sort((a, b) => a - b);
    for (let i of array) {
        let diff = Math.abs(n - i);
        if (diff < minDiff) {
            minDiff = diff;
            answer = i;
        }
    }
    return answer;
}